app/validation/validators.py

```diff
@@ -345,3 +345,34 @@ def format_date_string(field):
     return '{}-{:02d}-{:02d}'.format(int(field.data['year']),
                                      int(field.data['month']),
                                      day)
+
+
+class MutuallyExclusive():
```

Comment:
Even though its pretty self explanatory, would be nice to have a doc string here, like some of the others do